src/test/java/org/mastodon/revised/model/mamut/io/TestSaveLoadFeatures.java

```diff
@@ -0,0 +1,54 @@
+package org.mastodon.revised.model.mamut.io;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+
+import org.mastodon.pool.PoolCollectionWrapper;
+import org.mastodon.properties.DoublePropertyMap;
+import org.mastodon.revised.model.mamut.Model;
+import org.mastodon.revised.model.mamut.Spot;
+
+public class TestSaveLoadFeatures
```

Comment:
Is this incomplete or misnamed?

Reply:
To be removed. I could not get it right.